### DISCUSSION TITLE
Can I use Asymmetric Tiling in Comfyui?

### DISCUSSION THREAD

**Discussion by barepixels** [comfyanonymous/ComfyUI#1056] [Q&A]
For making a panoramic one need to make the Left and Right side of the image blend seamlessly.  The solution in a1111 is https://github.com/tjm35/asymmetric-tiling-sd-webui  Does this extension works in Comfyui and if so, how?

**Comment by ltdrdata** [comment]
Check this extension

https://github.com/BlenderNeko/ComfyUI_TiledKSampler

**Comment by barepixels** [comment]
Sadly [ComfyUI_TiledKSampler](https://github.com/BlenderNeko/ComfyUI_TiledKSampler) doesn't work.  It generates a 360 pano in each of the tile.  One can read how I successfully create 360 pano with a1111 here https://www.reddit.com/r/StableDiffusion/comments/15c3byu/comment/jtvcsw2/
